onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -379,7 +377,7 @@
 #endif
 
 #ifdef DEBUG_NODE_INPUTS_OUTPUTS
-    utils::DumpNodeOutputs(dump_context, op_kernel_context, p_op_kernel->Node(), session_state);
+    utils::DumpNodeOutputs(kernel_dump_context_, kernel_context_, kernel_.Node(), session_state_);
```

Comment:
>kernel_dump_context_, kernel_context_ [](http://example.com/codeflow?start=27&length=37)

nit: may be rename kernel_dump_context_ to dump_context_ to further differentiate from the 2nd parameter "kernel_context". It is ok that both session scope and kernel scope own a dump_context_.